db/migrate/20220202122417_add_parent_id_to_miq_requests.rb

```diff
@@ -0,0 +1,9 @@
+class AddParentIdToMiqRequests < ActiveRecord::Migration[6.0]
+  class MiqRequest < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+  
+  def change
+    add_column :miq_requests, :parent_id, :bigint
```

Comment:
Is this the id of another `miq_request` ?  I would have expected to see a `belongs_to :parent` relation in the `MiqReqeust` model over here https://github.com/ManageIQ/manageiq/pull/21681/files#diff-ecb22950a62fae5915a90e9d8ead61f3a7021f8ce173e9783f726784eb20b04d

Also usually good practice to use `add_reference` rather than `add_column` as that takes care of a lot of boiler-plate stuff for you like the type, index, etc...